tokio/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 1.24.0 (January 5, 2022)
+
+### Fixed
+ - rt: improve native `AtomicU64` support detection ([#5284])
+
+### Added
+ - rt: add configuration option for max number of I/O events polled from the OS
+   per tick (#5186)
```

Comment:
```suggestion
   per tick ([#5186])
```

Reply:
Thanks, I applied this change in the latest commit.